frontend/ui/menusorter.lua

```diff
@@ -86,7 +86,10 @@ function MenuSorter:sort(item_table, order)
                 if v then
                     if v.id == separator_id then
                         new_index = new_index - 1
-                        menu_table[order_id][new_index].separator = true
+                        -- ignore separator if the menu starts with it
+                        if new_index > 0 then
```

Comment:
@poire-z What do you suppose is more legible? `new_index > 0` or `new_index >= 1`?